### ISSUE TITLE
can't run example code.

### ISSUE THREAD

**Issue by feenes** [pavlov99/ajsonrpc#1]
Perhaps I don't understand the documentation or there is some other obvious issue, but I get an error when executing the example.

What I did.
```
$ python -V
Python 3.6.4
$ pip install ajsonrpc
...
 $ pip freeze | grep ajso
ajsonrpc==0.0.3
$ git clone https://github.com/pavlov99/ajsonrpc.git
...
$ cd ajsonrpc
$ async-json-rpc-server examples.methods
Traceback (most recent call last):
  File "/home/feenes/venv/py3/bin/async-json-rpc-server", line 10, in <module>
    sys.exit(main())
  File "/home/feenes/venv/py3/lib/python3.6/site-packages/ajsonrpc/scripts/server.py", line 66, in main
    module = importlib.util.module_from_spec(spec)
  File "<frozen importlib._bootstrap>", line 568, in module_from_spec
AttributeError: 'NoneType' object has no attribute 'loader'
$ cat examples/methods.py 
import asyncio

def echo(s='pong'):
    return s


def mul2(a, b):
    return a * b

async def say_after(delay, what):
    await asyncio.sleep(delay)
    return what
$
```

**Comment by pavlov99** [comment]
Thank you for reporting

**Comment by pavlov99** [comment]
Hi @feenes 

I took some time to refactor the library and it turned into a massive change, hence the delay. A new version of the package [1.0.0a0](https://pypi.org/project/ajsonrpc/1.0.0a0/) was released and it contains multiple fixes. I've also update the documentation and verified that examples work.

It took longer than expected. If this is still relevant, I would be grateful if you try this one more time!
